build.gradle

```diff
@@ -186,8 +186,7 @@ project(':kcbq-connector') {
 
         compile group: 'io.debezium', name: 'debezium-core',  version:'0.4.0'
 
-        compile group: 'org.apache.kafka', name: 'connect-api', version: '0.10.2.1'
-        compile group: 'org.apache.kafka', name: 'kafka-clients', version: '0.10.2.1'
```

Comment:
it looks to me like we use this in BigQuerySinkTask? https://github.com/wepay/kafka-connect-bigquery/blob/master/kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java#L44

Reply:
All Kafka dependencies get pulled in transitively as needed. The `SinkTask` use is via an interface included in the Connect APIs, so connectors should not be explicit in the version they depend on.

I regret letting this through as at the time I think we didn't realize we were exposing the transitive dependency on Kafka clients (and the dependence is *very* minimal). But for connectors, they should rely on the transitive dependency. It is already always guaranteed to be pinned to the same version given that the two projects are maintained in the same repo.

Reply:
but should kafka-clients be pulled in as compileOnly, like connect-api, or completely removed like it is now?

It also looks like the DBZ issue you linked that would have to be resolved first is now resolved. So, is this patch currently okay to merge or is there anything else on the dbz side that needs to be done beforehand?